lib/moon/components/select/multi_select.ex

```diff
@@ -163,9 +165,25 @@ defmodule Moon.Components.Select.MultiSelect do
         </PullAside>
       </FieldBorder>
       <:content>
-        <Popover.DefaultContent>
-          <Dropdown class="w-auto" id={"#{@id}-dropdown"} select_id={@id} {=@options} is_multi />
-        </Popover.DefaultContent>
+        {#if slot_assigned?(:default)}
```

Comment:
Sorry, no.

Currently the default implementation part has been copy-pasted into Partners and search added. Please add search directly to this component.